Packs/AnomaliSecurityAnalyticsAlerts/Integrations/AnomaliSecurityAnalyticsAlerts/AnomaliSecurityAnalyticsAlerts.py

```diff
@@ -220,15 +224,18 @@ def command_get_search_job_results(client: Client, args: dict) -> list[CommandRe
             if 'fields' in results_response and 'records' in results_response:
                 headers = results_response['fields']
                 records = results_response['records']
-                table_data = [dict(zip(headers, record)) for record in records]
+                combined_records = [dict(zip(headers, record)) for record in records]
```

Comment:
Can you add the type field to the zip output as well? 
What should be the value of combined_records can you give my an example?